modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsaMPCv2.ts

```diff
@@ -306,6 +308,19 @@ export class EcdsaMPCv2Utils extends BaseEcdsaUtils {
     ]);
     // #endregion
 
+    // #region reducing keychain data
```

Comment:
nit: do we really need these comments? I think creating separate function would be better